app/config/BackendConfig.scala

```diff
@@ -22,10 +22,9 @@ import uk.gov.hmrc.play.bootstrap.config.{RunMode, ServicesConfig}
 
 @Singleton
 class BackendConfig @Inject()(val servicesConfig: ServicesConfig,
-                              val environment:Environment,
+                              val environment: Environment,
                               val runModeConfiguration: Configuration,
-                              val runMode: RunMode
-                             ) {
+                              val runMode: RunMode) {
```

Comment:
I know it's nothing you've done on your PR, but it looks like this was missed from the play upgrade. The only thing we need to inject here is the services config - the rest are no longer needed. Don't need the mode thingy below either as that's all managed by the various app-configs now. Might be some places where you need to update runModeConfiguration to use servicesConfig.configuration instead.